package.json

```diff
@@ -20,10 +20,11 @@
   ],
   "dependencies": {
     "cookies": "^0.7.1",
-    "lodash": "^4.17.4",
-    "readable-stream": "^2.3.3"
+    "readable-stream": "^3.3.0"
```

Comment:
This needs to be ^2 not ^3

Reply:
I'm finding that as travis ci yells at me.  :)